tests/matrix_free/matrix_vector_hessians_cells.cc

```diff
@@ -183,20 +183,14 @@ test_hessians(const dealii::FE_Poly<dim>                    &fe,
             }
         }
     },
-    [&](
-      const auto &matrix_free, auto &dst, const auto &src, const auto &range) {
-      (void)matrix_free;
-      (void)dst;
-      (void)src;
-      (void)range;
-    },
-    [&](
-      const auto &matrix_free, auto &dst, const auto &src, const auto &range) {
-      (void)matrix_free;
-      (void)dst;
-      (void)src;
-      (void)range;
-    },
+    [&](const auto & /*matrix_free*/,
+        auto & /*dst*/,
+        const auto & /*src*/,
+        const auto & /*range*/) {},
+    [&](const auto & /*matrix_free*/,
+        auto & /*dst*/,
+        const auto & /*src*/,
+        const auto & /*range*/) {},
```

Comment:
This is in a test, so I don't care about this, but as a matter of general practice, we should probably just be passing empty `std::function` objects via `{}` in these places, rather than lambda functions that do nothing.